src/transformers/models/whisper/generation_whisper.py

```diff
@@ -865,7 +868,7 @@ def generate_with_fallback(
             if "decoder_attention_mask" in kwargs:
                 kwargs["decoder_attention_mask"] = torch.stack(new_decoder_attention_mask)
 
-        return seek_sequences, seek_outputs, should_skip, do_condition_on_prev_tokens
+        return seek_sequences, seek_outputs, should_skip, do_condition_on_prev_tokens, seek_outputs_short_form
```

Comment:
Would be cleaner **not** to return `seek_outputs_short_form` here, since we make a copy of `seek_outputs`, and then just do any necessary post-processing in `_postprocess_outputs` or the main body of generate